contrib/kube-prometheus/jsonnet/kube-prometheus/alerts/general.libsonnet

```diff
@@ -23,7 +23,7 @@
                 This alert is always firing, therefore it should always be firing in Alertmanager
                 and always fire against a receiver. There are integrations with various notification
                 mechanisms that send a notification when this alert is not firing. For example the
-                "DeadMansSnitch" integration in PagerDuty.
```

Comment:
This PagerDuty integration is actually called `DeadMansSnitch`, so this was correct.

Reply:
:man_facepalming: sorry about that, I assumed that was a typo, fixing